app/controllers/feedbacks/solicitations_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Feedbacks::SolicitationsController < ApplicationController
+  def create
+    @feedback = Feedback::Solicitation.new(feedback_params.merge(user: current_user))
+    @feedback.solicitation = Solicitation.find(feedback_params[:solicitation_id])
+    if @feedback.save!
+      @feedback_association_id = @feedback.solicitation.id
+      render 'feedbacks/create'
+    else
+      flash.alert = @feedback.errors.full_messages.to_sentence
+      redirect_back(fallback_location: root_path)
```

Comment:
On peut mettre les flash directement dans les redirect:
```suggestion
    redirect_back(fallback_location: root_path, alert:  @feedback.errors.full_messages.to_sentence)
```
(À un moment, il faudra qu’on fasse marcher les erreurs et les forms, on peut faire tellement mieux que juste afficher des flash.)